packages/editable-html/src/plugins/toolbar/toolbar.jsx

```diff
@@ -234,7 +246,7 @@ export class Toolbar extends React.Component {
             <IconButton
               aria-label="Delete"
               className={classes.iconRoot}
-              onMouseDown={e => plugin.deleteNode(e, node, value, onChange)}
+              onMouseDown={e => this.onDeleteMouseDown(e, plugin, node, value, onChange)}
```

Comment:
can we remove 1 step of indirection? and name the function describing the action instead of what triggered the action? eg:
`onMouseDown={e => this.onDeleteMouseDown(...)}` => `onMouseDown={ e => e.persist(); this.debouncedDeleteNode(...)}`

and can you add a test for the deleteNode action?